driver.go

```diff
@@ -332,8 +331,8 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 	for name, podmanClient := range d.podmanClients {
 		// Ping podman api
 		apiVersion, err := podmanClient.Ping(d.ctx)
-		attrPrefix := fmt.Sprintf("driver.podman.%s", name)
-		if name == "default" {
+		attrPrefix := fmt.Sprintf("driver.podman.%s", cleanUpSocketName(name))
+		if name == "default" || podmanClient.IsDefaultClient() {
```

Comment:
When we talked thru #371, the configuration we landed on was that an omitted name or the name "default" became the default client. Given that, the bug here is that we're not accounting for the `name == ""`, right? If we're setting the default client field correctly at config parsing time, can we just make this entire conditional like this?

```suggestion
		if podmanClient.IsDefaultClient() {
```

Reply:
I had to check the logic in `makePodmanClients()` but indeed this is the case. A socket without a name or with name default becomes the default socket. If none is found between all sockets, the first one becomes the default client. So checking `IsDefaultClient()` is sufficient as a condition.